### PR TITLE
Add skip layernorm[SiliconFlow]

### PR DESCRIPTION
**Pull request by MARD1NO**

### CODE REVIEW

File: src/flag_gems/__init__.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ def enable(lib=aten_lib):
     lib.impl("isinf", isinf, "CUDA")
     lib.impl("isnan", isnan, "CUDA")
     lib.impl("native_layer_norm", layer_norm, "AutogradCUDA")
+    lib.impl("skip_layer_norm", skip_layer_norm, "CUDA")
```
Comment (StrongSpoon) [inline src/flag_gems/__init__.py]: delete this line. ATen has no function named 'skip_layer_norm'. fused operators don't need registration here.

File: tests/flag_gems/op_accu_test.py [hunk 2]
```diff
@@ -523,6 +523,42 @@ def test_accuracy_layernorm(shape, dtype):
...
+def test_accuracy_skip_layernorm(shape, dtype):
+    M = shape[0]
+    N = shape[1]
+    layer_shape = [
+        N,
+    ]
+    inp = torch.randn(shape, dtype=dtype, device="cuda", requires_grad=False)
+    residual = torch.randn(shape, dtype=dtype, device="cuda", requires_grad=False)
+    weight = torch.randn(layer_shape, dtype=dtype, device="cuda", requires_grad=True)
+    bias = torch.randn(layer_shape, dtype=dtype, device="cuda", requires_grad=True)
```
Comment (StrongSpoon) [inline tests/flag_gems/op_accu_test.py]: argument requires_grad can be omitted

Comment (MARD1NO) [reply tests/flag_gems/op_accu_test.py]: Done

File: src/flag_gems/__init__.py (hunk 1 above)
Comment (MARD1NO) [reply src/flag_gems/__init__.py]: Done

**Pull request merged**